### PR TITLE
chore: improve badge verification and dep installs

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This improves the uds-badge verification task and dependency installs for GH

## Checklist before merging

- [ ] ADR proposed if making an architectural change to the repo
- [ ] Tests run, docs added or updated as needed

### CODE REVIEW

**Review by zachariahmiller — COMMENTED** [review]
Could you add a note in the docs on the `setup-ci` option?

**Comment by Racer159** [comment]
done

**Review by andrewg-xyz — APPROVED** [review]

File: docs/uds-packages/development/common-workflows.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Common GitHub Workflows
+
+This repository is designed to provide common workflows for downstream UDS Packages to consume and provide inputs into to run their CI checks.
+This helps to ensure consistency and helps improve these repos easily over time as Renovate watches this repo for changes.
```
Comment (zachariahmiller) [inline docs/uds-packages/development/common-workflows.md:4]: Maybe change this to either "This ensure consistency and..." or "This helps to ensure consistency and enables easily improving there repos over time..."

**Review by zachariahmiller — DISMISSED** [review]
Approving, I'll re-approve if you make adjustments based on my couple of comments, but i really dont feel strongly about them

**Review by zachariahmiller — APPROVED** [review]

**Pull request merged**